### ISSUE TITLE
empty-script triggers warning when passing a function as argument

### ISSUE THREAD

**Issue by ext** [alecxe/eslint-plugin-protractor#94]
The new `empty-script` rule (nice addition btw) triggers warnings when passing a function as argument. According to the [protractor documentation](https://www.protractortest.org/#/api?view=webdriver.WebDriver.prototype.executeScript) the passed function "will be converted to a string" automatically.

On the technical side it happens because the node type is `Identifier` and not `Literal` and it does not hold the `.value` property thus failing the test.

## Steps to reproduce

```js
function myClientScript(){
    /* this gets executed in the browser */
}

browser.executeScript(myClientScript);
```
## Expected results

No errors or warnings.

## Actual results

    5:9  warning  executeScript() called with an empty script  protractor/empty-script

**Comment by alecxe** [comment]
Oh, nice catch! My bad. Thanks for the report!

**Comment by alecxe** [comment]
Thanks for getting back to it and fixing @ext!
